packaging/nntrainer.spec

```diff
@@ -8,8 +8,37 @@
 %define         test_script $(pwd)/packaging/run_unittests.sh
 %define         gen_input $(pwd)/test/input_gen/genInput.py
 %define         support_data_augmentation_opencv 1
+
 %bcond_with tizen
 
+# dependency resolution
+%if 0%{?tizen_version_major}%{?tizen_version_minor} >= 65
+%define         capi_machine_learning_common	   capi-machine-learning-common
```

Comment:
is this tied with Tizen version or the nnstreamer version? nnstreamer version might be extendible to Debian packaging as well.

Reply:
Changed to query nnstreamer version for the extensibility.

Reply:
This doesn't work because of the way gbs or rpm parses and evaluates the shell scripts. (When querying for the nnstreamer version, nnstreamer is not available). I'll put instead a comment which nnstreamer version is expected for each tizen version

Reply:
I think nnstreamer won't work as nnstreamer would be installed after this file is processed based on `buildrequires`.

Reply:
@leemgs suggested to try `rpm -qi $PACKAGENAME` which does not require the package to be installed in the system. @zhoonit can you please try this?

Reply:
Unfortunately it doesn't work. I'll leave -qi as a comment